RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -861,20 +861,20 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
       }
       alpaka::syncBlockThreads(acc);
 
-      // Occupancy matrix for 0.8 GeV pT Cut
+      // Occupancy matrix for 0.8 GeV pT Cut, 99.9%
       constexpr int p08_occupancy_matrix[4][4] = {
-          {543, 235, 88, 46},  // category 0
-          {755, 347, 0, 0},    // category 1
-          {0, 0, 0, 0},        // category 2
-          {0, 38, 46, 39}      // category 3
+          {668, 271, 105, 59},  // category 0
+          {738, 310, 0, 0},     // category 1
+          {0, 13, 5, 0},        // category 2
```

Comment:
which kinematic regions are these 4 bins? (to follow up on the discussion during the meeting for why earlier zeroes are now not)

Reply:
![Screenshot 2025-01-24 at 6 06 58 PM](https://github.com/user-attachments/assets/00bf60c5-ac7d-46aa-971e-1d5a1618f6f4)

Reply:
this still requires parsing: what are the two middle elements out of 4 in the category 2?
Is Category 2 defined by radius and z range or by the disk and ring index in the endcap?

Reply:
<img width="1179" alt="Screenshot 2025-01-24 at 6 57 09 PM" src="https://github.com/user-attachments/assets/765b7d93-3ad8-443a-9764-fe7b2fc4d72d" />

Definition of Category 2: (module_layers >= 4) & (module_subdets == 5)
Definition of Eta ranges: 
    eta_numbers[module_eta < 0.75] = 0
    eta_numbers[(module_eta >= 0.75) & (module_eta < 1.5)] = 1
    eta_numbers[(module_eta >= 1.5) & (module_eta < 2.25)] = 2
    eta_numbers[(module_eta >= 2.25) & (module_eta < 3)] = 3